docsrc/users-guide/examples/VI as Sampler Inits.ipynb

```diff
@@ -4,13 +4,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Using Variational Estimates to Initialize the NUTS-HMC Sampler\n",
+    "## Using Estimates from Variational, Laplace, or Optimization Methods to Initialize the NUTS-HMC Sampler\n",
```

Comment:
we should rename this page altogether - "Initializing the NUTS-HMC sampler"

Reply:
Updated.